### PR TITLE
✨ Train Typeface Classifier

### PR DESCRIPTION
**Pull request by SphericalPotatoInVacuum**

### CODE REVIEW

File: src/training/simple.py [hunk 1]
```diff
@@ -36,7 +39,7 @@ def __init__(
     def train(self):
         self.model.train()
 
-        for inputs, target in self.train_dataloader():
+        for inputs, target in self.train_dataloader:
```
Comment (grenlayk) [inline src/training/simple.py:42]: Why don't we need these braces? Or why did we have them in the first place?....

File: src/training/simple.py [hunk 2]
```diff
@@ -48,7 +51,7 @@ def train(self):
             self.optimizer.step()
 
             if self.scheduler is not None:
-                self.scheduler.step()
+                self.scheduler.step(loss.item())
```
Comment (grenlayk) [inline src/training/simple.py:54]: Why do we pass `loss.item()` here?

File: src/training/simple.py (hunk 1 above)
Comment (SphericalPotatoInVacuum) [reply src/training/simple.py:42]: We don't need them because the dataloader is an object that we can iterate through. Why we had them in the first place is another question though, I think that got here from your first template draft, so that's kinda my bad for not proof checking that the code works before committing the previous version.

File: src/training/simple.py (hunk 2 above)
Comment (SphericalPotatoInVacuum) [reply src/training/simple.py:54]: Because some schedulers require losses to determine whether they should modify learning rate on this iteration. Howerver, if the scheduler doesn't need this, then it just ignores the loss IIRC.

File: src/training/simple.py (hunk 1 above)
Comment (grenlayk) [reply src/training/simple.py:42]: Got it.

**Review by nikoryagin — APPROVED** [review]
